docs/security_considerations.md

```diff
@@ -113,7 +113,7 @@ An attacker could use this information in an attempt to deduce the state of the
 **Mitigation and/or workaround:**
 
 The recommended mitigation as described in the user documentation is to use SSO integration. The default admin password
-should only be used for initial configuration and then [disabled](https://argoproj.github.io/argo-cd/operator-manual/user-management/#disable-admin-user)
+should only be used for initial configuration and then [disabled](https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#disable-admin-user)
```

Comment:
I think we can just link to `/operator-manual/user-management/#disable-admin-user` here.

Reply:
updated